scale-tests/deploy-dispatchers.py

```diff
@@ -1,3 +1,39 @@
+#!/usr/bin/env python3
+
+"""deploy-dispatchers.py
+
+Usage:
+    deploy-dispatchers.py [options] <num_dispatchers> <service_name_base> <output_file>
+
+Arguments:
+    num_dispatchers    number of dispatchers to deploy
+    service_name_base  DC/OS service name base
+    output_file        output file
+
+Options:
+    --cpus <n>                   number of CPUs to use per dispatcher [default: 1]
+    --docker-image <image>       custom Docker image to use
+    --enable-kerberos <bool>     enable Kerberos configuration [default: False]
+    --hdfs-config <url>          URL of the HDFS configuration files
+    --history-service <url>      URL of the Spark history service
+    --kdc-hostname <hostname>    the name or address of a host running a KDC
+    --kdc-port <port>            the port of the host running a KDC [default: 88]
+    --kerberos-realm <realm>     the Kerberos realm used to render the principal
+    --log-level <level>          log level [default: INFO]
+    --mem <n>                    amount of memory (mb) to use per dispatcher [default: 1024.0]
+    --options-json <file>        a file containing installation options in JSON format
+    --package-name <name>        name of the Spark package name [default: spark]
+    --role <role>                Mesos role registered by dispatcher [default: *]
+    --service-account <account>  Mesos principal registered by dispatcher
+    --service-secret <secret>    Mesos secret registered by dispatcher
+    --ucr-containerizer <bool>   launch using the Universal Container Runtime [default: True]
+    --user <user>                user to run dispatcher service as [default: root]
+
+"""
+
+from docopt import docopt
```

Comment:
Hi @sschneid sorry for the late review. For my own education: what are the benefits of `docopt` vs something more explicit like `argparse`?

Reply:
Are either of the two in the python standard library? Wouldn't mind avoiding pip whenever possible